gallery/visualtests/testUtils.js

```diff
@@ -189,6 +211,8 @@ module.exports = {
       dismissResultingDialog,
       disableLoadingSpinnerAnimation,
       scrollIntoView,
+      overrideDisplayTheme,
+      removeThemeEnabling,
```

Comment:
Let's keep the names of both of these consistent with the gallery-side functions that they invoke

Reply:
> Let's keep the names of both of these consistent with the gallery-side functions that they invoke

Updated here: [e3c5036](https://github.com/sonatype/sonatype-react-shared-components/pull/808/commits/e3c5036f7782f983fb251bbb0fa140836f3600d9)